### PR TITLE
generate references to load scripts in packages

### PR DESCRIPTION
**Pull request by dsyme**
Implements #3325

### CODE REVIEW

File: Paket.sln [hunk 1]
```diff
@@ -85,7 +85,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "commands", "commands", "{44
 		docs\content\commands\find-package-versions.md = docs\content\commands\find-package-versions.md
 		docs\content\commands\find-packages.md = docs\content\commands\find-packages.md
 		docs\content\commands\find-refs.md = docs\content\commands\find-refs.md
-		docs\content\commands\generate-include-scripts.md = docs\content\commands\generate-include-scripts.md
```
Comment (dsyme) [inline Paket.sln]: this file no longer exists, was a dud in the solution file

**Comment by dsyme** [comment]
A test is failing, I'm pretty sure it's unrelated as it fails for me on my local machine using the `.paket/paket.exe` used by this repo
```
Paket.IntegrationTests.InstallSpecs.#3062 install should use external azure functions v1 lock file from http Paket.IntegrationTests.dll 4 sec 225 ms
```
Error:
```
     Conflict detected:
      - External "azurefunctions-v1-packet.lock" requested package System.Buffers: 4.0
      - System.IO.Compression 4.3.0 requested package System.Buffers: >= 4.3

     Please try to relax some conditions or resolve the conflict manually (see http://fsprojects.github.io/Paket/nuget-dependencies.html#Use-exactly-this-version-constraint).
```

**Comment by matthid** [comment]
Yes please ignore that test, it is broken for some time and needs to be fixed by @forki ;)

**Comment by dsyme** [comment]
> Yes please ignore that test, it is broken for some time and needs to be fixed by @forki ;)

Thanks, labelled it as flaky

**Comment by dsyme** [comment]
Looks like Travis CI is pretty flaky too.  Can we move off Mono 5.4 to something a bit more modern?

**Comment by forki** [comment]
I'm fine with mono update. But usually just random other stuff breaks

Don Syme <notifications@github.com> schrieb am So., 5. Aug. 2018, 14:34:

**Pull request merged**